runtime-modules/forum/src/lib.rs

```diff
@@ -200,6 +200,31 @@ pub struct PollAlternative<Hash> {
     pub vote_count: u32,
 }
 
+/// Represents all poll alternative texts and vote count for each one
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug)]
+pub struct PollAlternativeInput {
+    /// alternative description
+    pub alternative_text: Vec<u8>,
+
+    /// Vote count for the alternative
+    pub vote_count: u32,
```

Comment:
Vote count should not be specified in the input. It should be always `0` for each alternative on poll thread creation.
I think input alternatives can be represented just as a simple `Vec<u8>` in `PollInput`

Reply:
Good catch. Done https://github.com/Joystream/joystream/pull/2531/commits/b8cb5726d2a221f8e38f14ff572b03933b821b66